x-pack/plugins/uptime/public/lib/alert_types/monitor_status.tsx

```diff
@@ -30,7 +35,17 @@ export const validate = (alertParams: unknown) => {
       },
     };
   }
-  if (isRight(decoded)) {
+  if (
+    (isRight(availabilityDecoded) && !availabilityDecoded.right.shouldCheckAvailability) ||
```

Comment:
```suggestion
    (isRight(availabilityDecoded) && !availabilityDecoded.right.shouldCheckAvailability) &&
```

Should be a boolean `&&` otherwise the state acts all wonky. We want to report an error if both availability AND status are false.

Reply:
I've improved this a bit. Have another look and LMKWYT.

https://github.com/elastic/kibana/pull/70284/commits/0517870331612b6be7ed3cb8ea25effdb84d78c6